### PR TITLE
Update README URLs based on HTTP redirects

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein

### GitHub Corrected URLs 
Was | Now 
--- | --- 
https://github.com/axefrog/SimpleBrowser | https://github.com/SimpleBrowserDotNet/SimpleBrowser 
https://github.com/niwibe/phantompy | https://github.com/niwinz/phantompy 
https://github.com/symfony/BrowserKit | https://github.com/symfony/browser-kit 

### Other Corrected URLs 
Was | Now 
--- | --- 
http://zombie.labnotes.org/ | http://zombie.js.org/

### CODE REVIEW

**Comment by ReadmeCritic** [comment]
Also consider removing http://www.envjs.com/ which 404s .. let me know if you want me to do that

**Comment by dhamaniasad** [comment]
@ReadmeCritic Sure, go ahead.

**Comment by ReadmeCritic** [comment]
@dhamaniasad removed

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Headless Browsers
 ================
 
-### A list of (almost) all headless web browsers in existence
+### A list of (almost) all headless enweb browsers in existence
```
Comment (dhamaniasad) [inline README.md]: Looks like a typo has been added.

**Comment by ReadmeCritic** [comment]
Oops

**Comment by ReadmeCritic** [comment]
@dhamaniasad ready

**Comment by dhamaniasad** [comment]
Thanks!

**Pull request merged**